trace/google/cloud/trace/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 Google LLC
+# Copyright 2017, Google LLC All rights reserved.
```

Comment:
From my reading of the OSPO guidelines, this was correct as it was, e.g.

`# Copyright 2017 Google LLC`

Unless you found another source that says something differently (very possible).